lib/topological_inventory/orchestrator/health_check.rb

```diff
@@ -0,0 +1,62 @@
+module TopologicalInventory
+  module Orchestrator
+    class HealthCheck
+      include Logging
+
+      attr_accessor :sources, :topology, :k8s
+
+      def initialize(sources, topology, k8s)
+        @sources = sources
```

Comment:
just a small suggestion, can you rename it to @sources_api/@topology_api to be consistent with the rest of code?